### DISCUSSION TITLE
Local release date for movies

### DISCUSSION THREAD

**Discussion by justkarmo** [trakt/showly#785] [Ideas / Feature Requests]
Hi! I really love this app, and having tried all of the ones which work with Trakt I can say that this is the best one. But for people outside of the US it might not be an option because it doesn't show local release dates for movies on the calendar and movie page. I know this is possible to do since both Cinexplore and Moviebase do it. Once in a while I re-download the app to see it this feature has been added since I'd love to switch to Showly, but I'm always a little bit disappointed. I know I'm not the only one that would switch to Showly if this option was added! Hope you can do it.

**Comment by ArthurTakase** [comment]
Hi, there,

I'm working on this option, which I also find very useful. Could you tell me what would be most relevant for you?

Here's the current state of my work to give you an idea of progress.

<div>
    <img src="https://github.com/michaldrabik/showly-2.0/assets/60554981/486e109f-cb69-484c-80e3-e00537f57a7f" width="200" alt="Screenshot 2">
<img src="https://github.com/michaldrabik/showly-2.0/assets/60554981/9b021ec8-736c-4ab9-b37e-69753ba04290" width="200" alt="Screenshot 2">
    <img src="https://github.com/michaldrabik/showly-2.0/assets/60554981/3e5a7dca-12e2-4c81-8ee0-dd1c2b3ca2fe" width="200" 
alt="Screenshot 3">
</div>


- "Drive away dolls" was released on 23 February in the US and 10 April in France.
- "Dune" was released on 21 October in the US and 15 September in France.

**Reply by justkarmo** [reply]
This looks great! You could merge the "release date" and "streaming services" section if that would make it easy for you and lighter for the app. I doubt that people would choose two different regions for those. And maybe you could add the region in parentesis with the release date on the movie page just like the US date; example: 15 Sept. 2021 (FR). Otherwise everything is perfect the way it is in your screenshoots. Keep up the good work!

**Reply by ArthurTakase** [reply]
You're right, merging the region and the release date is simpler.

As you suggested, I've added the country next to the date. There are several cases possible:
- If the local date is identical to the original release, both flags are displayed.
- If the local date is different, both dates are displayed, each with its flag.
- If no local date exists, this does not change the current app.

Did you have other things in mind or is this all good for you?

<div>
<img src="https://github.com/michaldrabik/showly-2.0/assets/60554981/b8747fcb-88fd-4009-b647-63cc712020d9" width="400px">
<img src="https://github.com/michaldrabik/showly-2.0/assets/60554981/64d81b5d-b2e0-4783-9c94-805d06e2a5a9" width="400px">
</div>

**Reply by justkarmo** [reply]
This looks perfect. Nothing to add. Great work 👍 Can't wait for the update

**Comment by ArthurTakase** [comment]
Open PR #800  to add the feature

**Comment by michaldrabik** [comment]
Thanks for the initiative! Right now I'm nowhere near PC but will be looking at it when I'm back.
